.github/workflows/test_python.yml

```
@@ -23,23 +27,24 @@ jobs:
           rm_cmd: "rmz" # NEW: Faster deletion
           rmz_version: "3.1.1" # NEW: Specify rmz version
 
-      - name: Set up Python
+      - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v4
```

actionlint (1.7.10)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)